contrib/cirrus/packer/fedora_setup.sh

```diff
@@ -22,7 +22,7 @@ install_ooe
 echo "Updating packages"
 ooe.sh sudo dnf update -y
 
-ooe.sh sudo dnf -y module install cri-o:1.13
+ooe.sh sudo dnf -y module install cri-o:1.17
```

Comment:
Do we already want to use 1.18 :question:

Reply:
baby steps :laughing: this PR doesn't even work properly right now lol